src/ignitus-Api/constants.js

```diff
@@ -1 +1,3 @@
-export const TESTIMONIAL_URL = 'https://ignitusrestapi.herokuapp.com/testimonials';
+const TESTIMONIAL_URL = 'https://ignitusrestapi.herokuapp.com/testimonials';
+
+export default TESTIMONIAL_URL;
```

Comment:
@darrylsepeda , ignore export default. We may have more constants.

Reply:
yes @outsourcegb 👍

Reply:
noted, will be changed on the next patch